eng/build.sh

```diff
@@ -74,6 +74,7 @@ usage()
   echo "  --gcc                      Optional argument to build using gcc in PATH (default)."
   echo "  --gccx.y                   Optional argument to build using gcc version x.y."
   echo "  --portablebuild            Optional argument: set to false to force a non-portable build."
+  echo "  --keepsymbols              Optional argument: set to true to keep symbols/debuginfo in generated binaries."
```

Comment:
The help message doesn't mention it takes an argument, but it seems to take true/false and reject nothingness. 😕 But it looks like this is also the case with other settings, so this PR appears to be consistent to me.

Reply:
Should I take a stab at fixing up the other settings too?